### ISSUE TITLE
IE8 Slow

### ISSUE THREAD

**Issue by DR9885** [select2/select2#787]
When having a list of 200+ items (Countries) it takes a few seconds before the list is rendered and then it delays before it re-positions. 

To eliminate the appearance of re-positioning in IE8, I changed the following:
<code>
this.updateResults(true);
if(this.dropdown[0] !== this.body().children().last()[0]) {
                this.dropdown.detach().appendTo(this.body());
}
</code>

To this instead:
<code>
if(this.dropdown[0] !== this.body().children().last()[0]) {
                this.dropdown.detach().appendTo(this.body());
}
this.updateResults(true);
</code>

However, this is still pretty slow... (Might need some optimizations for the html rendering).
